lambda/snapshots_tool_utils.py

```diff
@@ -373,3 +375,97 @@ def search_tag_copied(response):
 
     return False
 
+
+def get_all_automated_snapshots(client):
+    global _AUTOMATED_BACKUP_LIST
+    if len(_AUTOMATED_BACKUP_LIST) == 0:
+        response = paginate_api_call(
+            client,
+            'describe_db_cluster_snapshots',
+            'DBClusterSnapshots',
+            SnapshotType='automated',
+        )
+        _AUTOMATED_BACKUP_LIST = response['DBClusterSnapshots']
+
+    return _AUTOMATED_BACKUP_LIST
+
+
+def avail_db_snapshot(
```

Comment:
Another style comment: this function signature uses new lines for arguments where as the existing functions have a different format, lets maintain the same style as the original code